scripts/evaluate_mti.py

```diff
@@ -0,0 +1,56 @@
+"""
+Evaluate MTI tool from NLM on MeSH subset. The subset is automatically tagged
+by MTI through an email batch service and this scripts calculates performance
+by comparing with the ground truth.
+"""
+from argparse import ArgumentParser
+from pathlib import Path
+import pickle
+import json
+import csv
+
+from sklearn.metrics import classification_report
+
+
+def get_gold_tags(mesh_sample_data_path):
+    gold_tags = []
+    with open(mesh_sample_data_path) as f:
+        for line in f:
+            item = json.loads(line)
+            gold_tags.append(item["tags"])
+    return gold_tags
+
+def get_mti_tags(mti_output_path, mesh_tags):
+    mti_data = dict()
+    with open(mti_output_path) as f:
+        csvreader = csv.reader(f, delimiter="|")
+        for line in csvreader:
+            uid, tag, _, _ = line
```

Comment:
what is the other two returned fields?

Reply:
cuid and score. the first is a unique identifier of the tag, score is not a probability but a custom scoring function. why?

Reply:
I was just curious about why you are not using them. Makes sense.